build.gradle

```diff
@@ -36,7 +36,7 @@ buildscript {
     }
 }
 
-def pathApiDocs = "${rootDir}/modules/docs/arrow-docs/docs/docs/apidocs"
+def pathApiDocs = "${rootDir}/modules/docs/arrow-docs/docs/apidocs"
```

Comment:
I'll change this path afterwards, however, it won't impact because `apidocs` will be synchronized with the correspondent directory in S3.